src/OrchardCore/OrchardCore.Roles.Core/Services/DefaultSystemRoleNameProvider.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections.Frozen;
+using System.Runtime.CompilerServices;
+using OrchardCore.Environment.Shell;
+
+[assembly: InternalsVisibleTo("OrchardCore.Tests")]
```

Comment:
This can be added to the .csproj instead by using `InternalsVisibleTo`:

```xml
 <ItemGroup>
    <InternalsVisibleTo Include="OrchardCore.Tests" />
  </ItemGroup>
```

Reply:
done